aws_lambda_builders/__main__.py

```diff
@@ -131,6 +131,7 @@ def main():  # pylint: disable=too-many-statements
             architecture=params.get("architecture", X86_64),
             is_building_layer=params.get("is_building_layer", False),
             experimental_flags=params.get("experimental_flags", []),
+            build_in_source=params.get("build_in_source", None),
```

Comment:
default is None because some workflows are built in source by default and some are not. Planning to leave it up to each workflow to handle defaults.